src/ansys/hps/client/rms/models.py

```diff
@@ -34,683 +34,836 @@
+        description='Path to the shell script to template for the scheduler.',
+        title='Scheduler Script Override',
     )
     exclusive_default: Optional[bool] = Field(
         False,
-        description="Request the scheduler to hold the nodes exclusively for one request.",
-        title="Exclusive Default",
+        description='Request the scheduler to hold the nodes exclusively for one request.',
+        title='Exclusive Default',
     )
     distributed_default: Optional[bool] = Field(
         True,
-        description="Allow the scheduler to provide multiple machines to fulfill the request.",
-        title="Distributed Default",
+        description='Allow the scheduler to provide multiple machines to fulfill the request.',
+        title='Distributed Default',
     )
     num_cores_default: Optional[int] = Field(
         1,
-        description="Number of cores to request from the scheduler for a task.",
-        title="Num Cores Default",
+        description='Number of cores to request from the scheduler for a task.',
+        title='Num Cores Default',
     )
-    working_dir: Optional[str] = Field(
-        None, description="Working directory to use in the backend.", title="Working Dir"
+    shared_dir: Optional[str] = Field(
+        None, description='Working directory to use in the backend.', title='Shared Dir'
     )
     env: Optional[Dict[str, Any]] = Field(
-        None, description="Static environment variables needed for job execution.", title="Env"
+        None, description='Static environment variables needed for job execution.', title='Env'
     )
     process_runner: Optional[
         Union[ServiceUserProcessRunner, ProcessLauncherProcessRunner, RestLauncherProcessRunner]
     ] = Field(
-        {"plugin_name": "service_user_module"},
-        description="Process runner used to execute commands.",
-        discriminator="plugin_name",
-        title="Process Runner",
+        {'plugin_name': 'service_user_module'},
+        description='Process runner used to execute commands.',
+        discriminator='plugin_name',
+        title='Process Runner',
     )
     create_workdir: Optional[bool] = Field(
         True,
-        description="Create base and/or user-specific working directories at runtime.",
-        title="Create Workdir",
+        description='Create base and/or user-specific working directories at runtime.',
+        title='Create Workdir',
     )
-    use_templates: Optional[bool] = Field(
-        True,
-        description="Use the templated versions of the scripts and write them to the working directory.",
-        title="Use Templates",
+    use_local_scratch: Optional[bool] = Field(
+        False, description='To use local storage as working dir for jobs', title='Use Local Scratch'
     )
+    local_scratch_dir: Optional[str] = Field(
+        None,
+        description='Path to the local scratch directory to be used as jobs working dir',
```

Comment:
```suggestion
        description='Path to the local scratch directory to use as the job's working directory.',
```
I realized that you may have intended plural possessive (to use as the jobs' working directory). Please change if so.